src/collectors/rabbitmq/rabbitmq.py

```diff
@@ -144,6 +150,16 @@ def collect_health(self):
                              len(node_data['partitions']))
                 content = client.get_nodes()
                 self.publish('cluster.nodes', len(content))
+            if self.config['shovels']:
+                shovels = client.get_shovels()
+                for shovel in shovels:
+                    shovel_state = shovel["state"]
```

Comment:
Lines 156-161 could be replaced with the following:

```
shovel_state = 1 if shovel["state"] == "running" else 0
```

Reply:
Nice suggestion. Done!